### DISCUSSION TITLE
Informational resources for sliderule.h5.h5p datasets?

### DISCUSSION THREAD

**Discussion by gravinamike** [SlideRuleEarth/sliderule#318] [Troubleshooting] [answered]
Hello, I'm part of the Glacier Dynamics group at the Ohio State Byrd Polar and Climate Research Center.

We're interested in using the SlideRule Earth API to get access to GEDI L1, L2, L3 products, but only L4 appears to be currently available in the [SlideRule endpoints](https://urldefense.com/v3/__https:/slideruleearth.io/rtd/user_guide/SlideRule.html?highlight=gedi*assets__;Iw!!KGKeukY!yjpi62uOQx3QsQDyUq6w7-TD_UgviEqJBI95a2Co6_E6iec386C0SYG65tFIxiBe8wGQd-rGGpEOxkvjIw$).

We do see that there is an API for getting HDF5 files on the AWS servers, but the documentation for using it is a bit more sparse than that for the core module or the IceSat2 and GEDI plugins. We're having trouble determining whether the base products for GEDI exist or how to access them. I was wondering if there is a resource somewhere with info on datasets, resources and assets that can be obtained through sliderule.h5.h5p?

We're excited to see how this project develops in the coming months!

**Comment by dshean** [comment, marked as answer]
Hi @gravinamike.  Thanks for reaching out!

We are patiently waiting for public S3 access to the full archive of lower-level GEDI products to enable this capability.  We remain in a holding pattern, though additional questions/nudges from other users always helps.

In the meantime, we are developing the necessary SlideRule server and client functionality using sample L1/L2 GEDI data for select sites.
 
For now, you can use @tsutterley's tools for query, subsetting and local download GEDI data: https://github.com/tsutterley/lpdaac-gedi-subsetter.

**Reply by gravinamike** [reply]
Thanks @dshean , I'll check out @tsutterley 's subsetter and we'll be eagerly watching SlideRule Earth in the coming months!

**Comment by jpswinski** [comment]
@gravinamike In an effort to consolidate our points of contact with the community, we are transferring this discussion to our primary repository - https://github.com/ICESat2-SlideRule/sliderule.  Hopefully, this will make it it easier to monitor and respond to future discussion posts.
